flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherResourceCleanupTest.java

```diff
@@ -619,34 +618,6 @@ public boolean hasCleanJobResultEntry(JobID jobId) throws IOException {
         }
     }
 
-    @Test
-    public void testHABlobsAreNotRemovedIfHAJobGraphRemovalFails() throws Exception {
```

Comment:
why is this test no longer relevant?

Reply:
JobGraph removal and high availability cleanup are not depending on each other anymore.